src/libvirtApi/domain.js

```diff
@@ -890,6 +890,24 @@ export function domainSendNMI({
     return call(connectionName, objPath, 'org.libvirt.Domain', 'InjectNMI', [0], { timeout, type: 'u' });
 }
 
+async function domainSetXML(vm, option, values) {
+    const opts = { err: "message" };
+    if (vm.connectionName === 'system')
+        opts.superuser = 'try';
+
+    const args = [];
+    for (const key in values)
+        args.push(key + '="' + values[key].replaceAll('"', '\\"') + '"');
```

Comment:
There can be any arbitrary junk in the description, including more quotes, backspaces, etc. I think this isn't run through shell, so it shouldn't be that bad, but at the same time this means that we can't use `shlex.quote()` or similar. How much can we trust virt-xml's command line parser to not mess this up?

Reply:
Good catch, we need to also escape "\", but then we should be good... I'll check the virt-xml sources.

Reply:
For reference: https://github.com/virt-manager/virt-manager/blob/main/virtinst/cli.py#L1192

Reply:
Fixed.